File: 예제/Change/app/src/main/java/com/example/change/MainActivity.java
package com.example.change;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.text.Editable;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;
import android.widget.TextView;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity {

    TextView result;
    EditText first;
    Button button;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        first = (EditText) findViewById(R.id.first);
        result = (TextView) findViewById(R.id.result);
        button = (Button) findViewById(R.id.button);

        View.OnClickListener onClickListener = new View.OnClickListener() {

            @Override
            public void onClick(View view) {
                if (view.getId() == R.id.button) {
                    if (!first.getText().toString().equals("")) {
                        exchange();
                    } else {
                        Toast.makeText(getApplicationContext(), "값을 입력해주세요", Toast.LENGTH_SHORT).show();
                    }
                }
            }
        };
        button.setOnClickListener(onClickListener);
    }


            void exchange() {
                int count = Integer.parseInt(first.getText().toString());
                count *= 1121;
                String text = Integer.toString(count);
                result.setText(text);
            }
        };

File: 예제/Exchage/app/src/main/java/com/example/exchage/MainActivity.kt
package com.example.exchage
import android.os.Bundle
import android.widget.Button
import android.widget.EditText
import android.widget.TextView
import android.widget.Toast
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        var result = findViewById<TextView>(R.id.result)
        val result_button = findViewById<Button>(R.id.result_button)
        var before = findViewById<EditText>(R.id.doallar)
        var count = 0

        fun caculator(){
            var count = Integer.parseInt(before.text.toString())
            count *= 1121
            result.text = count.toString()
            Toast.makeText(this,"$count  원 입니다" ,Toast.LENGTH_SHORT).show();
            
        }

        result_button.setOnClickListener(){
            if(count >=0){
                caculator()
            }
            else if(count == 0 || count <= -1){
                var t1 =Toast.makeText(this, "정수를 입력해주세요", Toast.LENGTH_SHORT)
                t1.show()
                }
            }
        }
    }



